src/bilder/images/edxapp/packer_vars/xpro.pkrvars.hcl

```diff
@@ -0,0 +1,4 @@
+build_environment    = "mitxpro-ci"
+edx_platform_version = "open-release/maple.master"
+installation_target  = "xpro"
```

Comment:
Should this be `mitxpro`?

Reply:
No, we are using xpro for the naming of the environment but mitxpro for the OU tags.